js/app.js

```diff
@@ -0,0 +1,81 @@
+//person4------------------------------------------------------
+const person4 = new XMLHttpRequest();
+function person4Req() {
+  let person4Obj = JSON.parse(this.responseText);
+  document.getElementById("person4Name").innerHTML =
+    "Person 4: " + person4Obj.name;
+}
+person4.addEventListener("load", person4Req);
+person4.open("GET", "https://swapi.co/api/people/4/");
+person4.send();
+
+const person4Homeworld = new XMLHttpRequest();
+function person4HomeReq() {
+  let homeworld = JSON.parse(this.responseText);
+  document.getElementById("person4HomeWorld").innerHTML =
+    "HomeWorld: " + homeworld.name;
+}
+person4Homeworld.addEventListener("load", person4HomeReq);
+person4Homeworld.open("GET", "https://swapi.co/api/planets/1/");
+person4Homeworld.send();
+//person14------------------------------------------------------
+const person14 = new XMLHttpRequest();
+function person14Req() {
+  let person14Obj = JSON.parse(this.responseText);
+  document.getElementById("person14Name").innerHTML =
+    "Person 14: " + person14Obj.name;
+}
+person14.addEventListener("load", person14Req);
+person14.open("GET", "https://swapi.co/api/people/14/");
+person14.send();
+
+const person14Homeworld = new XMLHttpRequest();
+function person14HomeReq() {
+  let species = JSON.parse(this.responseText);
+  document.getElementById("person14Species").innerHTML =
+    "Species: " + species.name;
+}
+person14Homeworld.addEventListener("load", person14HomeReq);
+person14Homeworld.open("GET", "https://swapi.co/api/species/1/");
+person14Homeworld.send();
```

Comment:
Same with person 4, you should be calling this from the event handler for person 14.